aten/src/ATen/native/native_functions.yaml

```diff
@@ -286,6 +286,10 @@
   use_c10_dispatcher: full
   variants: function, method
 
+- func: conj_view(Tensor self) -> Tensor
```

Comment:
@bdhirsh asked why we named this separately as `conj_view`. Based on discussion with @anjali411 and co at https://github.com/pytorch/pytorch/pull/43270 we are seriously considering making `conj` a built-in